imap/cyr_dbtool.c

```diff
@@ -305,6 +305,11 @@ int main(int argc, char *argv[])
 
         fprintf(stderr, "\n");
         fprintf(stderr, "\n");
+        fprintf(stderr, "Options:\n");
+        fprintf(stderr, "  -c     convert database to named backend if not already\n");
+        fprintf(stderr, "  -M     use mailbox sort order\n");
```

Comment:
I'd like this to mention the word `improved_mboxlist_sort`, because that's what the rest of our documentation seems to call this thing, and it's probably useful to have a searchable keyword.

Something like:

```
        fprintf(stderr, "  -M     use \"improved_mboxlist_sort\" order\n");
```

maybe?

As I understand it, you need to whether your database is already in this order, and if it is you need to provide `-M` (or corruption might occur?), and if it isn't you need to not provided it (ditto).  Dragons etc.

Reply:
I have taken your suggestion wholesale.